README.md

```diff
@@ -10,15 +10,17 @@ Currently, this repo contains implementations of the rerankers for [CovidQA](htt
 
 ## Installation
 
-1. For pip, do `pip install pygaggle`. If you prefer Anaconda, use `conda env create -f environment.yml && conda activate pygaggle`.
+0. Clone this repo `git clone git@github.com:castorini/pygaggle.git`
```

Comment:
It seems confusing to me that we need to clone AND do a pip install.. maybe move the clone instruction to the eval section, so it is needed only by users who want to run the evaluation scripts?

Reply:
FWIW, we should distinguish between (1) just using pygaggle, in which case `pip` is sufficient, and (2) developing/evaluating, in which we need to clone the entire repo etc,.

Reply:
Yea you are both right. Will change and clarify.